communication/moderation.md

```diff
@@ -220,6 +220,19 @@ umbrella.
 - [Zoom Guidelines](./zoom-guidelines.md)
 - [Moderators](./moderators.md#zoom)
 
+### Process for Adding a Moderator
+
+This is the workflow for adding a moderator. 
+
+- Moderator applies by filing an issue in the kubernetes/community repo
+- Moderator get's 2 approvals from sponsors
+- Add the person to their respective moderation tool:
+  - Slack - Add them as a slack workspace admins (instructions pinned on #slack-admins-private): Invite them to #slack-admins-private, #slack-admins, #slack-log, #slack-reports and #slack-invites
+  - Discuss - Change their permission to moderator
+  - Google group - PR the person into the correct yaml file in kubernetes/k8s.io
```

Comment:
This is only true things that `@kubernetes.io` properties. k-dev etc are still managed out of band through the UI